app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    # Split the command to extract source and destination
+    _, src, dest = command.split()
```

Comment:
The `command.split()` assumes that the command string is always correctly formatted with exactly three parts. Consider adding error handling to manage cases where the command might not be formatted as expected.